popup/popup.ts

```diff
@@ -31,6 +31,11 @@ function updateAppState (newState) {
 
 function updateNotifications () {
     const notificationContainer = $("#notifications");
+    if (appState.notifications.length) {
```

Comment:
It's more "typescripty" to do an explicit number comparison at this point rather than falling back to a javascript truthy comparison.
Not a deal breaker but I think it would be better if it were:
`(appState.notifications.length > 0)`